### ISSUE TITLE
About the train of Feedback Policy: a bug

### ISSUE THREAD

**Issue by gouyinghong** [OpenDriveLab/CLOVER#5]
Hi, I'm trying to run `train_calvin.sh`. When running to `FeedbackPolicy/train/train_utils.py`, but I encountered an error.
![image](https://github.com/user-attachments/assets/aaf4e243-e174-4e82-adf0-6cb6aed0fa7a)
I find that the output of model which represents the state of gripper is not in [0, 1]
![image](https://github.com/user-attachments/assets/ee419d76-9df0-4a4a-aab1-5109f28bec03)

**Comment by retsuh-bqw** [comment]
I've modified the loss func to `binary_cross_entropy_with_logits`, should have solved the issue

**Comment by gouyinghong** [comment]
> I've modified the loss func to `binary_cross_entropy_with_logits`, should have solved the issue

It works. Thanks for your help!
